docs/API.AMQP.md

```diff
@@ -1413,27 +1295,15 @@ These statistics include:
 
 Statistics are returned as a JSON-object with each key being the snapshot's timestamp.
 
-
-**Topic:** ``admin..getAllStats``
-
-**Exchange name:** ``amq.topic``
-
-**replyTo queue metadata:** Required.
+**replyTo queue header:** Required.
 
 **Query:**
 
 ```javascript
 {
-  /*
-  Optional
-  */
-  clientId: <Unique session ID>,
-
-  /*
-  Optional: Kuzzle will forward this field in its response, allowing you
-  to easily identify which query generated the response you got.
-  */
-  requestId: <Unique query ID>
+  controller: 'admin',
+  action: 'getAllStats',
+  collection: ''
```

Comment:
for global route, I think it is best to remove the ``collection`` argument altogether